test/integration/test_utils.mjs

```diff
@@ -34,9 +37,8 @@ function loadAndWait(filename, selector, zoom, pageSetup) {
       });
 
       let url = `${global.integrationBaseUrl}?file=/test/pdfs/${filename}`;
-      if (zoom) {
-        url += `#zoom=${zoom}`;
-      }
+      url += `#zoom=${zoom ?? "page-fit"}`;
```

Comment:
If the zoom argument is always set, I think this can be a `const` instead?
```javascript
const url =
  `${global.integrationBaseUrl}?file=/test/pdfs/${filename}#zoom=${zoom ?? "page-fit"}`;
```